libs/vue-route-guard/project.json

```diff
@@ -4,9 +4,23 @@
   "sourceRoot": "libs/vue-route-guard/src",
   "targets": {
     "build": {
+      "executor": "@nrwl/workspace:run-commands",
```

Comment:
Nice!
I've been playing with it a bit so that we don't need to duplicate the path to dist folder and the following modification seems to work nice
```
    "build": {
      "executor": "@nrwl/workspace:run-commands",
      "options": {
        "parallel": false,
        "commands": [
          {
            "command": "nx build-app vue-route-guard --dist={args.dist}/dist"
          },
          {
            "command": "cp 'libs/vue-route-guard/package.json' '{args.dist}/package.json'",
            "forwardAllArgs": false
          }
        ],
        "dist": "dist/libs/vue-route-guard"
      }
    },
    "build-app": {
      "executor": "nx-vue3-vite:build-app"
    },
```